src/database/operations/serviceQueryAuditLogs/createServiceQueryAuditLog.ts

```diff
@@ -1,18 +1,20 @@
 import TinyPg from 'tinypg';
 import { AuthContext } from '../../../types';
 
-export const createDbOperationAuditLog = async (
+const createServiceQueryAuditLog = async (
 	db: TinyPg,
 	authContext: AuthContext | null,
 	createValues: {
 		queryName: string;
 		queryParameters: object;
 	},
 ): Promise<void> => {
-	await db.sql('dbOperationAuditLogs.insertOne', {
+	await db.sql<object>('serviceQueryAuditLogs.insertOne', {
```

Comment:
`@typescript-eslint` was happier with this than the previous one after making the imports more consistent.